generic/secrets/security/detected-aws-access-key-id-value.txt

```diff
@@ -1,5 +1,11 @@
 # ruleid: detected-aws-access-key-id-value
-AKIAIOSFODNN7EXAMPLE
+AKIAIOSFODNN7EAXMPLE
 
 # ruleid: detected-aws-access-key-id-value
+ASIAIOSFODNN7EAXMPLE
```

Comment:
is it intended to have 2 the same tests?

Reply:
The first one is AKIA, the second one is ASIA